src/viewer.ts

```diff
@@ -392,15 +391,18 @@ class Viewer {
                 name: 'skyboxPrefilter' + i,
                 width: sizes[i],
                 height: sizes[i],
-                // @ts-ignore TODO type property missing from pc.Texture
                 type: pc.TEXTURETYPE_RGBM,
                 addressU: pc.ADDRESS_CLAMP_TO_EDGE,
                 addressV: pc.ADDRESS_CLAMP_TO_EDGE,
                 fixCubemapSeams: true,
                 mipmaps: false
             });
-            // @ts-ignore
-            pc.reprojectTexture(device, cubemaps[1] || skybox, prefilter, specPower[i], 4096);
+
+            pc.reprojectTexture(cubemaps[1] || skybox, prefilter, {
+                numSamples: 4096,
+                specularPower: specPower[i] === undefined ? 1 : specPower[i]
```

Comment:
This looks weird to me @slimbuck. How come the first element in the `specPower` array is `undefined`?

Reply:
Yup, thanks.

Reply:
Done. 👍